tests/test_core.py

```diff
@@ -2046,3 +2085,92 @@ def test_scaper_with_short_background():
         audio1, sr = soundfile.read(wav1_file.name)
         audio2, sr = soundfile.read(wav2_file.name)
         assert np.allclose(audio1, audio2)
+
+
+def test_clipping_and_normalization():
+
+    for sr in [16000, 44100]:
+        sc = scaper.Scaper(10, FG_PATH, BG_PATH, random_state=0)
+        sc.sr = sr
+        sc.ref_db = -20
+
+        sc.add_event(
+            label=('const', 'siren'),
+            source_file=('choose', []),
+            source_time=('uniform', 0, 10),
+            event_time=('const', 0),
+            event_duration=('const', 10),
+            snr=('const', 20),
+            pitch_shift=None,
+            time_stretch=None)
+
+        # extreme clipping
+        sc_extreme = scaper.Scaper(10, FG_PATH, BG_PATH, random_state=0)
+        sc_extreme.sr = 16000
+        sc_extreme.ref_db = -20
```

Comment:
It might be stronger to iterate over a few `ref_db` values instead of just one?

Reply:
Well it only clips if SNR is high enough with respect to red_db, meaning I'd have to also change the SNR value, meaning I'm not sure that would actually test anything different?